### DISCUSSION TITLE
Nextcloud v27 doesn't start

### DISCUSSION THREAD

**Discussion by tkupka** [alexbelgium/hassio-addons#876] [Q&A] [answered]
After upgrading to version 27.0.0 nextcloud won't start due to PHP error:
```
This version of Nextcloud is not compatible with PHP>=8.2.<br/>You are currently running 8.2.6.Please install nextcloud first
...done

**Comment by alexbelgium** [comment]
Hi, please create an issue instead to follow the template (including full logs).
It's strange for me it works. Could you please paste full logs ? I think that's because it wasn't updated (probably auto-update addon option isn't on)

Edit : same as https://github.com/linuxserver/docker-nextcloud/issues/313 on upstream repo

**Comment by alexbelgium** [comment, marked as answer]
The best way for you actually would be to restore the previous version, go in the administration options, update webui to 27.0, then perform update again.

Your nextcloud version is probably too old for the automatic updater, which would explain that it didn't update despite me enabling automatic updates a couple versions ago

**Comment by tkupka** [comment]
Thanks for your help. I had v 24.0.3 thus I had to downgrade and update to v27.0.0. This was caused that I disabled auto update option. Now it's running fine.

**Comment by alexbelgium** [comment]
Great, glad you were able to downgrade. 

From a user perspective, would you have preferred that the disable_updates did not exist so that you would be forces to always have an up-to-date system and avoid such breaking? I was hesitating to remove it

**Reply by tkupka** [reply]
I would prefer it to have it manually controlled in case of troubles or any other reasons. When it's enabled by default then it's fine I would say. If I remember well I disabled it because of the DB backup warning and it was unclear to me that it would cause this issue in the future. So some warning in the doc would be nice :-)

**Reply by alexbelgium** [reply]
Hi, ok then I'll keep it as an addon option, but with a big red disclaimer that things will go wrong if it's not done manually ;-)

**Comment by alexbelgium** [comment]
I've changed the addon logic : disable_update will prevent apps from updating, not nextcloud. This will avoid this issue in the future

**Reply by tkupka** [reply]
Yep That's what I thought so I disabled it ;-)
You are doing cool work!
